packages/ng-table-builder/src/table-builder.component.ts

```diff
@@ -763,4 +802,10 @@ export class TableBuilderComponent<T>
             simpleRenderedKeys: this.templateParser.templateKeys!
         };
     }
+
+    private listenExpandChange(): void {
+        this.headerTemplate?.expandedChange.pipe(takeUntil(this.destroy$)).subscribe((_: boolean): void => {
```

Comment:
`( (): void => this.updateTableHeight() )`

Reply:
:peka:

Reply:
fix